src/mpid/ch4/src/mpidig_pt2pt_callbacks.c

```diff
@@ -140,7 +140,6 @@ static int recv_target_cmpl_cb(MPIR_Request * rreq)
          * that came from CH4 (e.g. MPIDI_recv_safe) */
         MPIR_Request *sigreq = MPIDIG_REQUEST(rreq, req->rreq.match_req);
         sigreq->status = rreq->status;
-        MPIR_Request_add_ref(sigreq);
```

Comment:
The reference was already added here - https://github.com/hzhou/mpich/blob/ad47e0cc80037eedd899754fa761d235b4911e94/src/mpid/ch4/src/mpidig_request.h#L64